Owncloud iOs Client/DataBase/Queries/ManageUsersDB.h

```diff
@@ -22,76 +22,83 @@
 
 @interface ManageUsersDB : NSObject
 
+
+typedef enum {
```

Comment:
The enum should be on the UserDto because it is a class more simple than the manage and the UserDto will be always imported when we need to use the "sorting"